### PR TITLE
Express PoC

### PR DESCRIPTION
**Pull request by crgold**
Pull request to merge branch, containing stable version of the backend/frontend framework, into main

### CODE REVIEW

File: src/pages/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+      <div className="App">
+        <header className="App-header">
+          <h1 className="App-title">Welcome to React</h1>
+        </header>
+        // Render the newly fetched data inside of this.state.data 
+        <p className="App-intro">{this.state.data}</p>
+      </div>
+    );
+  }
+}*/
```
Comment (Sergprieto) [inline src/pages/Signup.jsx:179]: Can delete this older version of the signup page.

File: src/routes/routes.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import Login from '../pages/Login';
+import Signup from '../pages/Signup';
+import { Switch, Route } from 'react-router-dom';
+
+function Routes() {
+    return (
+        <Switch>
+                <Route exact path='/' component = {ifAuth(DiceLanding)} />
+                <Route path='/login' component = {Login} />
+                <Route path='/signup' component = {Signup} />
```
Comment (Sergprieto) [inline src/routes/routes.js:14]: Have to check with the team to determine what we want the default page to be when they enter our site. As it stands right now users will automatically be asked to login (or will already be logged in).

**Review by Sergprieto — COMMENTED** [review]
Everything looks great, just want to confirm with the team what we want to do about the default page that the user sees.

**Review by akclifto — APPROVED** [review]
Looks great, heroku app functional.

**Pull request merged**